main/src/main/scala-2/com/typesafe/scalalogging/LoggerImpl.scala

```diff
@@ -0,0 +1,85 @@
+package com.typesafe.scalalogging
```

Comment:
this file is essentially a rename, with minor changes `Scala2LoggerMacro` to avoid a name clash